synapse/lib/snap.py

```diff
@@ -368,6 +368,7 @@ async def nodesByProp(self, full):
             for layr in self.layers:
                 genr = layr.liftByProp(None, prop.name)
                 async for node in self._joinStorGenr(layr, genr):
+                    # TODO should these type of filters yield?
```

Comment:
no todos in mainline code

Reply:
err fixmes.

Reply:
There is a question about how many results one could have that whiff in the continue block or not. that's really the question about yielding here.